library/psa_crypto_pake.c

```diff
@@ -453,7 +445,13 @@ static psa_status_t psa_pake_output_internal(
         if( operation->state == PSA_PAKE_STATE_SETUP ) {
             status = psa_pake_ecjpake_setup( operation );
             if( status != PSA_SUCCESS )
+            {
+                mbedtls_platform_zeroize( operation->password, operation->password_len );
```

Comment:
Is this really necessary? This is wrapped by a function that calls `abort()` on errors. (Also, it's clearly unnecessary if we make sure `psa_pake_ecjpake_setup()` always zeroizes, both on success and on failure, as I suggest above).